db/migrations/20200901135641_create_cat_claw.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+CREATE TABLE maker.cat_claw
+(
+    id        SERIAL PRIMARY KEY,
+    header_id INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    address_id BIGINT  NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    log_id    BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    msg_sender BIGINT NOT NULL REFERENCES addresses (id) ON DELETE CASCADE,
+    rad      NUMERIC,
```

Comment:
Why is this rad and not litter?

Reply:
It's `rad` since that is what the argument for the `claw` method: https://github.com/makerdao/dss/blob/master/src/cat.sol#L178